src/main/webapp/WEB-INF/tags/instructor/studentList/studentsTable.tag

```diff
@@ -0,0 +1,41 @@
+<%@ tag description="instructorStudentList - Students table per course" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="course" type="teammates.ui.template.InstructorStudentListStudentsTableCourse" required="true" %>
+<%@ attribute name="index" required="true" %>
+<c:choose>
+    <c:when test="${course.courseArchived}">
+        <c:set var="PANEL_TYPE" value="panel-default" />
+    </c:when>
+    <c:otherwise>
+        <c:set var="PANEL_TYPE" value="panel-info" />
+    </c:otherwise>
+</c:choose>
```

Comment:
I believe it's also possible to do this with a one-liner:
`<c:set var="PANEL_TYPE" value="${course.courseArchived ? 'panel-default' : 'panel-info'} />`
Feel free to ignore this if you prefer this, though, it's just a matter of style.